packages/block-library/src/pullquote/edit.js

```diff
@@ -124,12 +129,13 @@ class PullQuoteEdit extends Component {
 
 		return (
 			<>
-				<figure style={ figureStyles } className={ figureClasses }>
-					<blockquote
+				<Figure style={ figureStyles } className={ figureClasses }>
+					<BlockQuote
 						style={ blockquoteStyles }
 						className={ blockquoteClasses }
 					>
 						<RichText
+							identifier="value"
```

Comment:
Identifier and textAlign are only used by native code right?

Reply:
True for `textAlign`. But `identifier` also exists in web: https://github.com/WordPress/gutenberg/blob/master/packages/block-editor/src/components/rich-text/index.js#L118